MapleServer2/Tools/GameCommandActions.cs

```diff
@@ -20,12 +20,28 @@ public static void Process(GameSession session, string command)
             string[] args = command.ToLower().Split(" ", 2);
             switch (args[0])
             {
+                case "fi":
+                    session.SendNotice($"map: {session.Player.MapId} instance:{session.Player.InstanceId}");
+                    break;
```

Comment:
Is this needed in master? Or was it just used for debugging?

Reply:
perhaps not, but I thought I would leave it in, it seemed useful to me to have a command to check your instance and map id, as they will remain to be variables that will need to be checked.

Reply:
if that's the case, perhaps we should have the map command display the instance you're at. Currently, if you don't feed it an id, it'll display the map ID you're in.

Reply:
yea, totally true, Why did I not just do that haha.